internal/api/impl/migrate/mapping.cuepart

```diff
@@ -1,13 +1,37 @@
 import (
-	// regexp is not used at the current stage in this file. It is used when merging the different cuepart file together. This is a way to shared the import across the different files
     "regexp"
-    "strings"
+    "strings" 
     "math"
 )
 
+#mapping: {
+    // mapping table for the unit attribute (key = grafana unit, value = perses equivalent)
+    unit: {
+        ms: "Milliseconds"
+        s: "Seconds"
+        m: "Minutes"
+        h: "Hours"
+        d: "Days"
+        percent: "Percent"
+        percentunit: "PercentDecimal"
+        bytes: "Bytes"
+        decbytes: "Bytes"
+    }
+    // mapping table for the calculation attribute (key = grafana unit, value = perses equivalent)
+    calc: {
+        first: "First"
+        firstNotNull: "First"
+        last: "Last"
+        lastNotNull: "LastNumber"
+        mean: "Mean"
+        total: "Sum"
+    }
+}
+#defaultCalc: "Last"
+
 kind: "Dashboard",
 metadata: {
-    name: strings.Replace(#grafanaDashboard.title, " ", "", -1)
+    name: #grafanaDashboard.uid
```

Comment:
yes this is actually a proposal, the reasons behind are
- the UID can be set manually
- the UID can easily be changed before migrating the dashboard to Perses format (on the UI it's among the last fields of the list)
- I think it makes it easier for people that would like to compare before and after, because the URLs of both tools would use the same dashboard reference

Reply:
sounds good to me